test/user.spec.js

```diff
@@ -0,0 +1,51 @@
+const request = require('supertest');
+const app = require('../app');
+const { User } = require('../app/models');
+const { USER } = require('./constants');
+
+describe('User Creation', () => {
+  it('Service responds with 200 when user is created correctly', () =>
```

Comment:
this name is not properly. it seems you are testing only the status code. you should tell that user is created properly when params are ok or something like that
we dont use the term "service" here.